pod-operation/src/components/high_voltage_system.rs

```diff
@@ -0,0 +1,31 @@
+use tracing::debug;
+
+use rppal::gpio::{Gpio, OutputPin};
+
+pub struct HighVoltageSystem {
+	pin: OutputPin,
+}
+
+const PIN_HIGH_VOLTAGE_SYSTEM: u8 = 20;
```

Comment:
Nitpick: apologies for not originally specifying, but following suit from last year, could we rename this to `PIN_CONTACTOR_RELAY` to allude to the physical operation?

Reply:
ofc